src/Grids/conformal_cubed_sphere_face_grid.jl

```diff
@@ -60,7 +60,7 @@ struct ConformalCubedSphereFaceGrid{FT, TX, TY, TZ, A, R, Arch} <: AbstractHoriz
     end
 end
 
-function ConformalCubedSphereFaceGrid(arch::AbstractArchitecture = CPU(),
+function ConformalCubedSphereFaceGrid(architecture::AbstractArchitecture = CPU(),
```

Comment:
We need a new name for this grid --- it's important features are that it's on the sphere, but that it's coordinate lines are not geographic (so its not a "lat-lon grid").

Also like all our grids it is orthogonal and makes the thin shell approximation (the vertical grid is "extruded").

Perhaps `OrthogonalSphericalShellGrid` or something.